lib/code_corps/github/adapters/task.ex

```diff
@@ -26,6 +28,8 @@ defmodule CodeCorps.GitHub.Adapters.Task do
     task
     |> Map.from_struct
     |> MapTransformer.transform_inverse(@mapping)
+    |> Map.delete("created_at")
     |> Map.delete("number")
+    |> Map.delete("updated_at")
```

Comment:
Maybe a function `clear_autogenerated_github_keys`, which removes all of these keys, for added explicitness and self-documenting?